source/org/zfin/profile/Organization.java

```diff
@@ -328,33 +176,44 @@ public String getAbbreviationOrder() {
         return name;
     }
 
-    public Set<Person> getMemberList() {
-        return memberList;
-    }
-
-    public void setMemberList(Set<Person> memberList) {
-        this.memberList = memberList;
-    }
-
     @Override
     public String getEntityType() {
         return getType();
     }
 
+    public String getEmailIfVisible() {
+        if (emailPrivacyPreference == null) {
+            return email;
+        }
+        return emailPrivacyPreference.getEmailIfVisibleOrEmptyString(email, ProfileService::getCurrentSecurityUser);
```

Comment:
It looks like this method is duplicated in Person, Lab and Organization. Could you write it once and re-use in all three entities?

Reply:
This is done with the latest commit